scraper/company_scrapper.go

```diff
@@ -0,0 +1,140 @@
+package scraper
+
+import (
+	"bufio"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"regexp"
+	"strings"
+
+	"github.com/google/go-querystring/query"
+)
+
+// SerpAPI response struct visit: https://serper.dev/playground
+type SerpAPIResponse struct {
+	Organic []struct {
+		Link string `json:"link"`
+	} `json:"organic"`
+}
+
+func ReadCompanyNames(filepath string) ([]string, error) {
+	file, err := os.Open(filepath)
+	if err != nil {
+		return nil, err
+	}
+	defer file.Close()
+
+	var companyNames []string
+	scanner := bufio.NewScanner(file)
+	for scanner.Scan() {
+		companyNames = append(companyNames, scanner.Text())
+	}
+
+	if err := scanner.Err(); err != nil {
+		return nil, err
+	}
+
+	return companyNames, nil
+}
+
+func GetSearchResults(client *http.Client, companyName string) (string, error) {
+
+	os.Setenv("SERPAPI_KEY", "0eb5aec35da6593d1993b1573558d3b5f8b0a37c")
```

Comment:
exposed security keys. use config library https://github.com/spf13/viper 
we do not set default variables for certain configs like security keys

Reply:
This has been fixed